package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eq-author-graphql-schema",
-  "version": "0.17.0",
```

Comment:
Looks like this is a few versions behind now. Worth publishing a new minor version (e.g. 0.18.0 once composite answers are introduced?